src/components/MainTest/MainTest.test.tsx

```diff
@@ -14,6 +14,6 @@ import MainTest from "./MainTest";
  */
 describe("MainTest", () => {
   test("renders MainTest", () => {
-    render(<MainTest text="Hello World" />);
+    //render(<MainTest text="Hello World" />);
```

Comment:
We should not comment out tests. ( Just kidding :D )

Reply:
But you told me to, boss.